From foo@baz Tue Apr  9 12:12:43 2002
To: Greg KH <greg@kroah.com>
Date: 25 Feb 2005 14:36:10 -08:00
From: benh@kernel.crashing.org
Subject: [PATCH] PCI: Apple PCI IDs update

please sent that to Andrew/Linus in your next batch for after 2.6.11,
those new IDs will be needed for support of the new iMac G5.

The changes to pci.ids match the changes already submitted to the web
database.

From: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


diff -Nru a/include/linux/pci_ids.h b/include/linux/pci_ids.h
--- a/include/linux/pci_ids.h	2005-03-02 23:04:18 -08:00
+++ b/include/linux/pci_ids.h	2005-03-02 23:04:18 -08:00
@@ -862,6 +862,9 @@
 #define PCI_DEVICE_ID_APPLE_KEYLARGO_I	0x003e
 #define PCI_DEVICE_ID_APPLE_K2_ATA100	0x0043
 #define PCI_DEVICE_ID_APPLE_K2_GMAC	0x004c
+#define PCI_DEVICE_ID_APPLE_SH_ATA      0x0050
+#define PCI_DEVICE_ID_APPLE_SH_SUNGEM   0x0051
+#define PCI_DEVICE_ID_APPLE_SH_FW       0x0052
 #define PCI_DEVICE_ID_APPLE_TIGON3	0x1645
 
 #define PCI_VENDOR_ID_YAMAHA		0x1073
